src/ui/templates/insurance/account/suspended.njk

```diff
@@ -0,0 +1,15 @@
+{% extends 'index.njk' %}
+
+{% block pageTitle %}
+  {{ CONTENT_STRINGS.PAGE_TITLE }}
+{% endblock %}
+
+{% block content %}
+
+  <h1 class="govuk-heading-xl" data-cy="heading">{{ CONTENT_STRINGS.PAGE_TITLE }}</h1>
+
+  <p class="govuk-body" data-cy="body">{{ CONTENT_STRINGS.BODY }} </p>
+
+  <a class="govuk-button" href="#" data-cy="reactivate-account-link-button">{{ CONTENT_STRINGS.BUTTONS.REACTIVATE_ACCOUNT }}</a>
```

Comment:
could we put the href in CONTENT_STRINGS as # so we only need to change it there then?

Reply:
Good idea, but when this link is implemented it'll have to come from the variables passed from the controller as we'll need to pass dynamic query parameter. We do this in other account related pages